ci-operator/config/openshift/machine-api-provider-openstack/openshift-machine-api-provider-openstack-master.yaml

```diff
@@ -71,6 +71,16 @@ tests:
       CUSTOM_FEATURE_FLAGS_ENABLED: MachineAPIProviderOpenStack
       TEST_SUITE: experimental/reliability/minimal
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-proxy
+  optional: true
+  steps:
+    cluster_profile: openstack-vh-mecha-central
+    env:
+      BASE_DOMAIN: shiftstack.devcluster.openshift.com
+      CONFIG_TYPE: minimal
+      CUSTOM_FEATURE_FLAGS_ENABLED: MachineAPIProviderOpenStack
+      TEST_SUITE: experimental/reliability/minimal
+    workflow: openshift-e2e-openstack-proxy
```

Comment:
you don't need that, it's defined in the workflow already.

Reply:
I assume you refer to the test suite. OK. Incidentally, I don't think `minimal` should be defined in the workflow, but nopt a topic for this PR.